package.json

```diff
@@ -1,27 +1,32 @@
 {
   "name": "ng-feedhenry",
   "version": "0.2.53",
-  "main": "./src/ngFH.js",
+  "main": "./dist/ngFH.js",
   "description": "AngularJS friendly FeedHenry SDK wrapper.",
   "repository": "feedhenry-staff/ngFeedHenry",
   "scripts": {
-    "test": "make test"
+    "clean": "rm -r dist test/bundle.js",
+    "build": "npm run lint && ./node_modules/.bin/browserify -e ./src/ngFH.js -o ./dist/ngFH.js",
+    "debugbuild": "./node_modules/.bin/browserify -e ./src/ngFH.js -o ./test/bundle.js",
+    "prepublish": "npm run build",
+    "lint": "./node_modules/.bin/eslint .",
+    "test": "npm run debugbuild && ./node_modules/karma/bin/karma start"
   },
   "authors": [
     "Evan Shortiss"
   ],
   "license": "MIT",
   "devDependencies": {
+    "angular": "~1.5.8",
+    "angular-mocks": "~1.5.8",
     "browserify": "~4.1.8",
-    "karma": "^0.12.21",
+    "eslint": "~3.9.1",
+    "fh-mocks": "0.1.1",
+    "karma": "~1.2.0",
     "karma-chai": "~0.1.0",
-    "karma-mocha": "~0.1.9",
-    "karma-chrome-launcher": "~0.1.3",
-    "linelint": "0.0.2",
-    "jshint": "^2.5.3",
-    "lintspaces-cli": "0.0.4",
-    "fh-mocks": "~0.0.5",
-    "karma-mocha-reporter": "~0.3.1"
+    "karma-chrome-launcher": "~2.0.0",
+    "karma-mocha": "~1.0.1",
+    "karma-mocha-reporter": "~2.0.4"
   },
   "dependencies": {
```

Comment:
I think these can go in "devDependencies" now since when someone does an `npm install` they get the `dist` folder which already has these :+1: